sdk/typescript/src/client/client.ts

```diff
@@ -440,10 +440,10 @@ export class SuiClient {
 			transactionBytes = await transactionBlock.build({ client: this });
 		}
 
-		const { signature, bytes } = await signer.signTransactionBlock(transactionBytes);
+		const { signature } = await signer.signTransactionBlock(transactionBytes);
 
 		return this.executeTransactionBlock({
-			transactionBlock: bytes,
+			transactionBlock: transactionBytes,
```

Comment:
Do you need to make this change? `bytes` is already being encoded always in the sign as a base64 string, seems like using that here might be better?

Reply:
good catch...  I thought we would decode the string to bytes not encoded to bytes to a string, got this backwards